Developed in-house at Oracle
Will file a bug upstream asking this to be an option for configure

diff -rupN old/utils/Makefile.am new/utils/Makefile.am
--- old/utils/Makefile.am	2016-02-12 11:48:32.389775435 -0800
+++ new/utils/Makefile.am	2016-02-12 11:51:08.007216490 -0800
@@ -48,7 +48,7 @@ all_sasl_static_libs = ../lib/.libs/libs
 sbin_PROGRAMS = @SASL_DB_UTILS@ @SMTPTEST_PROGRAM@ pluginviewer
 EXTRA_PROGRAMS = saslpasswd2 sasldblistusers2 testsuite testsuitestatic smtptest pluginviewer
 
-noinst_PROGRAMS = dbconverter-2
+noinst_PROGRAMS = dbconverter-2 testsuite
 
 if NO_SASL_DB_MANS
 man_MANS = 
